README.md

```diff
@@ -35,7 +35,8 @@ Parameters (specified as one object passed into hot-shots):
 * `prefix`:      What to prefix each stat name with `default: ''`
 * `suffix`:      What to suffix each stat name with `default: ''`
 * `globalize`:   Expose this StatsD instance globally. `default: false`
-* `cacheDns`:    Cache the initial dns lookup to *host* `default: false`
+* `cacheDns`:    Cache the initial dns lookup to *host* , only used
+  when protocol is `uds`, `default: false`
```

Comment:
Clarify when this is used, as I realize it has always been this way when going through the code